multi/imxrt-multi/imxrt-multi.c

```diff
@@ -81,13 +82,23 @@ static void multi_dispatchMsg(msg_t *msg)
 		case id_spi2:
 		case id_spi3:
 		case id_spi4:
+			spi_handleMsg(msg, id);
+			break;
+
 #ifdef TARGET_IMXRT1170
 		case id_spi5:
 		case id_spi6:
-#endif
 			spi_handleMsg(msg, id);
 			break;
 
+		case id_cm4_0:
```

Comment:
I'm not sure if it's good idea to demux messages based on custom multi ID. Why not use `msg->i.openclose.oid.id` instead? Anyway read() & write() messages will be not delivered because multi ID is not set (it probably applies to TRNG driver as well).

Reply:
In mtDevCtl we don't have this field - we only have msg.i/o.raw, so we need to pass oid/id by ourself. Oid in common msg field is planned on API change

Reply:
Ok. I see what you mean. It worked by a miracle, fixed